src/Components/Cover.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
You import react everywhere is there any way to import it single place and available all over the app.